src/test/Simulation.t.sol

```diff
@@ -723,4 +992,55 @@ contract SimulationTests is DSTest {
         takeLpDaiEth(auctionId, amount, auctionPrice, minProfit);
         assertGe(dai.balanceOf(bobAddr), minProfit);
     }
+
+    function takeSteCRV(
+        uint256 auctionId,
+        uint256 amt,
+        uint256 max,
+        uint256 minProfit
+    ) private {
+        vat.hope(steCRVClipAddr);
+
+        uint24 poolFee = 3000;
+        bytes memory path = abi.encodePacked(wethAddr, poolFee, daiAddr);
+        bytes memory data = abi.encode(
+            edAddr,         // to
+            steCRVJoinAddr, // gemJoin
+            minProfit,      // minProfit
+            0,              // coinIndex (0 - eth)
+            path,           // path
+            cropManagerAddr // manager
+        );
+
+        steCRVClip.take(auctionId, amt, max, edAddr, data);
+    }
+
+    function testTakeSteCRVNoProfit() public {
```

Comment:
Ensure the auction was completed by comparing `steCRVClip.count()` before and after `take()`

Reply:
Done in c388ca4